diff --git a/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx b/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
index 6a94382..4835f2e 100644
--- a/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
+++ b/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
@@ -120,7 +120,7 @@ function isInlineStyleTransform(transform: any): boolean {
 }
 
 function hasInlineStyles(style: StyleProps): boolean {
-  return Object.keys(style).some((key) => {
+  return Object.keys(style ?? {}).some((key) => {
     const styleValue = style[key];
     return (
       isSharedValue(styleValue) ||
@@ -139,7 +139,7 @@ function extractSharedValuesMapFromProps(
     if (key === 'style') {
       const styles = flattenArray<StyleProps>(props.style ?? []);
       styles.forEach((style) => {
-        for (const [key, styleValue] of Object.entries(style)) {
+        for (const [key, styleValue] of Object.entries(style ?? {})) {
           if (isSharedValue(styleValue)) {
             inlineProps[key] = styleValue;
           } else if (
